appveyor.yml

```diff
@@ -24,4 +24,4 @@ install:
 build: off
 
 test_script:
-  - cmd: grunt test
+  - cmd: grunt
```

Comment:
If appveyor rely on npm test by default, then this could also be cleaned up further